packages/exporter-trace-otlp-http/src/platform/node/util.ts

```diff
@@ -76,9 +77,16 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
     });
   });
 
+  req.setTimeout(exporterTimeout, function requestTimeout () {
+    req.destroy();
+  });
 
-  req.on('error', (error: Error) => {
-    onError(error);
+  req.on('error', (error: Error | any) => {
+    if (error.code === 'ECONNRESET') {
```

Comment:
you may get `ECONNRESET` for several reasons, not just request timeout (e.g. server decides to end the connection...). see  [HTTP docs](https://nodejs.org/dist/latest-v16.x/docs/api/http.html) for some examples.

Reply:
I can add an additional check to see if a [request was called with `destroy` ](https://nodejs.org/dist/latest-v16.x/docs/api/http.html#requestdestroyed) so:

```js
if (error.code === 'ECONNRESET' & req.destroyed)
```

Reply:
`req.destroyed` is only available for node >=14.1.0.
But I think it should be not that hard to store this info somewhere in exporter at the time `req.destroy` is called.

Reply:
Got it. I'll store it in a variable.